code-challenges.test.js

```diff
@@ -21,7 +21,17 @@ var secretCodeWord1 = "lackadaisical"
 // Expected output: "l4ck4d41s1c4l"
 var secretCodeWord2 = "gobbledygook"
 // Expected output: "g0bbl3dyg00k"
-
+describe("when string with consonents and vowels is called", () => {
+  it("returns a string with the vowels coded to integers", () => {
+    const stringWithConsonets = str.split(' ')
```

Comment:
you'll need to declare a function outside of the tests which will hold the logic you're trying to test. Then reference it within the tests. 

Down with the for-loops! 😈